### DISCUSSION TITLE
Marketing Team Weekly Meeting Planning 2022-03-23

### DISCUSSION THREAD

**Discussion by nalininsbs** [WordPress/Marketing-Team#20] [General]
Please add your suggestions for this week's Global Marketing Meeting on 23 March 2022. If you have any updates for the meeting, you can also add them to this discussion card.

This card is to assist the notetakers too. The script document used to save time during the meeting will be added to this card. Please add notes to the bottom of this document. Thank you

**Comment by eidolonnight** [comment]
Scripting for the meeting drafted in this [Google doc](https://docs.google.com/document/d/1jA267WcL5O1zuEKD_G2M-XWcBY3H9iL5xo14FuI2khQ/edit#heading=h.8mc8sa2h1r38)
